### PR TITLE
fix #289672, fix #272546 and more: fix some strings

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
* Fix #289672: "Reset '$STRING' value" strings reference non-existent $STRING
* Fix #272546: Baritone Oboe should be named Bass Oboe in English (see also #4474),
   "Hautbois Baryton" in French, "Bariton Oboe" in German, "Oboe Bajo" in Spanish.

Plus:

* Add (back) units for sp and pt
* Clarify transposition settings in staff properties dialog
* Fix a reset button's accessibilty info to match the label
* Quote buttons/keys in tours
* Barré to Barre
* Fix capitalization (see also #5102)
* Don't translate debugger (which is not visible in RELEASE mode)
* Sforzato to Accent in shortcuts (see https://musescore.org/en/node/290297)
* Some whitespace and punctuation fixes
* Fixing some error messages
* Removing a redundant word
* Disambiguations for various occurences of "None" which may require different translatations.
* Simplify some strings with URLs for translators
* Don't translate script recorder (which is not visible in RELEASE mode)

In a separate commit (removed from this PR now, possibly will create a new PR for it, once this here got merged):

* Change the shortcut for sforzato to accent
   Not sure we really want this though, as it invalidates an existing shortcut settings. We may get away with this by documenting in the release notes that this shortcut gets lost during an upgrade and needs to get reinstated manually (or via a factory reset).

### CODE REVIEW

File: libmscore/cmd.cpp [hunk 1]
```diff
@@ -3652,7 +3652,7 @@ void Score::cmd(const QAction* a, EditData& ed)
             { "add-staccato",               [this]{ addArticulation(SymId::articStaccatoAbove);                 }},
             { "add-tenuto",                 [this]{ addArticulation(SymId::articTenutoAbove);                   }},
             { "add-marcato",                [this]{ addArticulation(SymId::articMarcatoAbove);                  }},
-            { "add-sforzato",               [this]{ addArticulation(SymId::articAccentAbove);                   }},
+            { "add-accent",                 [this]{ addArticulation(SymId::articAccentAbove);                   }},
```
Comment (Jojo-Schmitz) [inline libmscore/cmd.cpp]: I wonder whether this change should be done, it'd invalidate existing shortcuts

File: mscore/shortcut.cpp [hunk 2]
```diff
@@ -701,9 +701,9 @@ Shortcut Shortcut::_sc[] = {
       {
          MsWidget::SCORE_TAB,
          STATE_NORMAL | STATE_NOTE_ENTRY,
-         "add-sforzato",
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: I wonder whether this change should be done, it'd invalidate existing shortcuts.
Also requires changes to the 3 shortcuts*.xml we distribute

File: mscore/shortcut.cpp [hunk 3]
```diff
@@ -701,9 +701,9 @@ Shortcut Shortcut::_sc[] = {
...
-         QT_TRANSLATE_NOOP("action","Sforzato"),
-         QT_TRANSLATE_NOOP("action","Toggle sforzato"),
+         "add-accent",
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: I wonder whether this change should be done, it'd invalidate existing shortcuts.
Also requires changes to the 3 shortcuts*.xml we distribute

File: mscore/data/shortcuts.xml [hunk 4]
```diff
@@ -221,7 +221,7 @@
     <seq>Shift+O</seq>
     </SC>
   <SC>
-    <key>add-sforzato</key>
+    <key>add-accent</key>
```
Comment (dmitrio95) [inline mscore/data/shortcuts.xml]: Will this change work correctly with existing users' shortcuts files? Shortcut file will contain Shift+V shortcut which references `add-sforzato` action which doesn't exist anymore, so the shortcut may happen to be lost at all.

Actually I don't know the answer, perhaps the new default will somehow take a precedence. But that probably needs testing.

Comment (Jojo-Schmitz) [reply mscore/data/shortcuts.xml]: An existing shortcut for 'add-sforzato' will simply get ignored

Comment (Jojo-Schmitz) [reply mscore/data/shortcuts.xml]: An existing **changed** shortcut for 'add-sforzato' will simply get ignored.
But the default shortcut Shift+V will continue to work.

Comment (Jojo-Schmitz) [reply mscore/data/shortcuts.xml]: An existing **changed** shortcut for 'add-sforzato' will simply get ignored.
But the default shortcut Shift+V will continue to work (and map to add-accent), after a factory reset.
Without a factory reste, the old existing shortucst would get used and not have have this shortcut set up.

Comment (Jojo-Schmitz) [reply mscore/data/shortcuts.xml]: On start I see "Debug: cannot find shortcut <add-sforzato> (...\MuseScore\mscore\shortcut.cpp:4167, static void Ms::Shortcut::load())". The shortcut `add-sforzato` will simple get ignored. The user would either need to set up a new shortcutr (e.g. "Shift+V" again) `for add-accent` or take a factory reset.

**Pull request merged**